### PR TITLE
docs: update design migration docs for Beta 2

### PR DESCRIPTION
**Pull request by laurenmrice**
#### Changelog

**New**

- Added images to each section.

**Changed**

- Updated content to reflect what will be released in Beta 2

----

**Testing**
- Go to `Migrating` --> `Guide` ---> `Design` tab

### CODE REVIEW

File: src/pages/migrating/guide/design.mdx [hunk 1]
```diff
@@ -72,51 +73,166 @@ Beta kit is available in the White and Gray 100 themes.
...
+
+<Row>
+<Column colLg={8}>
+
+![Example of a tooltip.](images/tooltip-usage-1.png)
+
+</Column>
+</Row>
+
+#### Toggletip
```
Comment (kingtraceyj) [inline src/pages/migrating/guide/design.mdx:131]:
```suggestion
### Toggletip
```
should toggle tip have a h3 heading?

**Review by kingtraceyj — CHANGES_REQUESTED** [review]

**Review by kingtraceyj — APPROVED** [review]
good to go! 💥

**Review by andreancardona — APPROVED** [review]
yay!

**Review by jeanservaas — COMMENTED** [review]
Looks great!!!!

Quick comment about the images... I'd go ahead and make these 6 column squares (with a gutter) so they don't seem so crammed.

image


I'm on the fence about this one... wondering if more white space would help here because the margins are so tight on token tabs (i.e. going 6 column squares which each of these images too) 

image

I would mean showing more of the UI screen, not enlarging it... so I can't really mock it up, but I think that image would benefit from a little more air.

File: src/pages/migrating/guide/design.mdx [hunk 2]
```diff
@@ -72,55 +73,160 @@ Beta kit is available in the White and Gray 100 themes.
...
+
+<Row>
+<Column colLg={8}>
+
+![Example of toast notifications stacking.](images/05_toast_context_608.gif)
+
+</Column>
+</Row>
+
+### Popover
```
Comment (sstrubberg) [inline src/pages/migrating/guide/design.mdx:95]: @joshblack are these components in good enough shape that we should be giving guidance on their usage?

Comment (joshblack) [reply src/pages/migrating/guide/design.mdx:95]: @sstrubberg I think we can for the beta if people understand that this stuff is subject to change.

**Review by sstrubberg — APPROVED** [review]

**Review by jeanservaas — APPROVED** [review]
Looks great!

**Pull request merged**